cpp/unreal_plugins/SimulationController/Source/SimulationController/ImitationLearningTask.h

```diff
@@ -44,26 +42,18 @@ class ImitationLearningTask : public Task {
     void actorHitEventHandler(AActor* self_actor, AActor* other_actor, FVector normal_impulse, const FHitResult& hit_result);
 
     // Get a list of the different initial and goal positions the agent is expected to navigate to.
-    // Positions are in the format "init.X, init.Y, init.Z, goal.X, goal.Y, goal.Z".
+    // Positions are in the format "start_location.X, start_location.Y, start_location.Z, end_location.X, end_location.Y, end_location.Z".
```

Comment:
Need to include `scene_id` in this comment.

Reply:
Updated this.